gtest/src/manager.rs

```diff
@@ -475,8 +478,62 @@ impl ExtManager {
             .into_parts()
             .0;
 
-        // The `metawasm` macro knows how to decode this as a tuple
-        let mut mapping_code_payload = argument.unwrap_or_default();
+        let metadata =
```

Comment:
It would be great to add some comments for these steps of getting function signatures.